### PR TITLE
Add doc for paddle.i1 and paddle.i1e document.;test=docs_preview

### PR DESCRIPTION
**Pull request by LyndonKong**
### PR types
New features

### Chnages
Add docs for paddle.i1 and paddle.i1e.

### References
code link: https://github.com/PaddlePaddle/Paddle/pull/53210
rfc link: https://github.com/PaddlePaddle/community/pull/489

### CODE REVIEW

File: docs/api/paddle/Tensor_cn.rst [hunk 1]
```diff
@@ -2533,3 +2533,21 @@ vander(x, n=None, increasing=False, name=None)
 返回类型：Tensor
 
 请参考 :ref:`cn_api_paddle_vander`
+
+i1(x, name=None)
+:::::::::
+用于将输入的 Tensor 计算第一类一阶修正贝塞尔函数。
+返回：返回第一类一阶修正贝塞尔函数对应输出 Tensor。
```
Comment (sunzhongkai588) [inline docs/api/paddle/Tensor_cn.rst:2572]: 上方加空行，否则会和上一行连在一起
image

File: docs/api/paddle/Tensor_cn.rst [hunk 2]
```diff
@@ -2533,3 +2533,21 @@ vander(x, n=None, increasing=False, name=None)
...
+
+返回类型：Tensor
+
+请参考 :ref:`cn_api_paddle_tensor_i1`
+
+i1e(x, name=None)
+:::::::::
+用于将输入的 Tensor 计算第一类指数缩放的一阶修正贝塞尔函数。
+返回：返回第一类指数缩放的一阶修正贝塞尔函数对应输出 Tensor。
```
Comment (sunzhongkai588) [inline docs/api/paddle/Tensor_cn.rst:2582]: 上方加空行

File: docs/api/api_label [hunk 3]
```diff
@@ -1218,4 +1218,5 @@ paddle.get_cuda_rng_state	.. _api_paddle_get_cuda_rng_state:
 paddle.flops	.. _api_paddle_flops:
 paddle.load	.. _api_paddle_load:
 paddle.vander	.. _api_paddle_vander:
-
+paddle.i1	.. _api_paddle_i1:
+paddle.i1e	.. _api_paddle_i1e:
```
Comment (sunzhongkai588) [inline docs/api/api_label]: 标签和文档的rst文件第一行进行统一，改成
`.. _api_paddle_tensor_i1:`
`.. _api_paddle_tensor_i1e:`

File: docs/api/paddle/i1_cn.rst [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+.. py:function:: paddle.i1(x, name=None)
+对于给定 ``x`` 计算其每个元素的第一类一阶修正贝塞尔曲线函数，其中输入 ``x`` 大小无特殊限制。返回第一类一阶修正贝塞尔函数对应输出 Tensor。
+
+.. math::
+    I_1(x)=\frac{\left(\text { x }_{i}\right)}{2} * \sum_{k=0}^{\infty} \frac{\left(\text{x}_{i}^{2} / 4\right)^{k}}{(k !) *(k+1) !}
+参数
+::::::::::
+    - **x** (Tensor) – 输入是一个多维的 Tensor，它的数据类型可以是 float32，float64。
+    输入 `Tensor`，其形状为任意维度的 Tensor。数据类型为 float32 或 float64。
```
Comment (sunzhongkai588) [inline docs/api/paddle/i1_cn.rst]: - 这一行内容的作用是？感觉在重复描述

- 且这一行开头需要和 `**x** (Tensor)` 对齐，否则会显示错误

image

File: docs/api/paddle/i1e_cn.rst
Comment (sunzhongkai588) [inline docs/api/paddle/i1e_cn.rst:1]: 和` i1 `文档存在同样的问题

**Review by sunzhongkai588 — COMMENTED** [review]
请修复，并且参考中文模板 https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/dev_guides/api_contributing_guides/api_docs_guidelines_cn.html#zhongwenmuban  加上空行

**Comment by LyndonKong** [comment]
Fix bug in our new commit @sunzhongkai588

File: docs/api/paddle/i1e_cn.rst [hunk 5]
```diff
@@ -0,0 +1,25 @@
+.. _cn_api_paddle_tensor_i1e:
+
+i1
```
Comment (sunzhongkai588) [inline docs/api/paddle/i1e_cn.rst]: 名称错了，是i1e

File: docs/api/paddle/i1e_cn.rst [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+-------------------------------
+
+.. py:function:: paddle.i1(x, name=None)
```
Comment (sunzhongkai588) [inline docs/api/paddle/i1e_cn.rst]: 同理，应该是i1e

**Comment by LyndonKong** [comment]
We have resolved confilits and fix the typo.

**Review by sunzhongkai588 — APPROVED** [review]
LGTM

**Pull request merged**